src/auto_archiver/archivers/youtubedl_archiver.py

```diff
@@ -30,6 +30,8 @@ def configs() -> dict:
             "end_means_success": {"default": True, "help": "if True, any archived content will mean a 'success', if False this archiver will not return a 'success' stage; this is useful for cases when the yt-dlp will archive a video but ignore other types of content like images or text only pages that the subsequent archivers can retrieve."},
             'allow_playlist': {"default": False, "help": "If True will also download playlists, set to False if the expectation is to download a single video."},
             "max_downloads": {"default": "inf", "help": "Use to limit the number of videos to download when a channel or long page is being extracted. 'inf' means no limit."},
+            "cookies_from_browser": {"default": None, "help": "optional browser for ytdl to extract cookies from, can be one of: brave, chrome, chromium, edge, firefox, opera, safari, vivaldi, whale"},
```

Comment:
there is a `"choices": ['...']` key for the config values but as it stands I don't think it allows for `None` to be selected, so this is just all right but a small improvement to consider in the more general config.py logic.

Reply:
True. yt-dlp also allows tuples for  (`browsername:profilename`) but I thought that might be a bit too complicated to include here (for the average user). Happy to extend this to allow either `browser` or `browser:profilename` if you think it is worth it.

Ref: https://github.com/yt-dlp/yt-dlp/wiki/FAQ#how-do-i-pass-cookies-to-yt-dlp